definitions.yaml

```diff
@@ -12,6 +12,36 @@ definitions:
         description: |
           A machine readable [response code](https://github.com/giantswarm/api-spec/blob/master/details/RESPONSE_CODES.md) like e. g. `INVALID_CREDENTIALS`
 
+  # App Catalogs Response
+  V4AppCatalogsResponse:
+    type: array
+    items:
+      type: object
+      properties:
+        description:
+          description: A description of the catalog.
+          type: string
+        logoURL:
+          description: A URL to a logo representing this catalog.
```

Comment:
```suggestion
          description: An URL to a logo representing this catalog.
```
nitpick

Reply:
I think "A URL" is correct?

I realise I didn't apply this consistently later in the definition though. So will change it all to `a url`

https://www.a-or-an.com/a_an/url

Reply:
ok good